### ISSUE TITLE
Accessibility: jupyter.org/governance/standing_committees_and_working_groups.html

### ISSUE THREAD

**Issue by Eshy10** [jupyterhub/outreachy#85]
### What page is this for?

https://jupyter.org/governance/standing_committees_and_working_groups.html

### WAVE accessibility report

1. Empty Form Label
2. Missing or Invalid Language
3. Multiple Form Label
4. Empty Button

<img width="1440" alt="Screenshot 2022-10-18 at 03 22 53" src="https://user-images.githubusercontent.com/52670459/196320697-4d34e8b3-c90e-448c-aeb0-e2336e3b08c7.png">

### The HTML element to be changed

```html
1. Empty Form Label:
<label for="toctree-checkbox-1">
<i class="fas fa-chevron-down"></i>
</label>

<label for="toctree-checkbox-2">
<i class="fas fa-chevron-down"></i>
</label>

<label for="toctree-checkbox-3">
<i class="fas fa-chevron-down"></i>
</label>

<label for="__navigation" class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
<span class="headerbtn__icon-container">
<i class="fas fa-bars"></i>
</span>
</label>

<label for="__page-toc" class="headerbtn headerbtn-page-toc">
<span class="headerbtn__icon-container">
<i class="fas fa-list"></i>
</span>
</label>

2.  Missing or Invalid Language:
<html pwa-launched="true" pwa-extension-id="npnbdojkgkbcdfdjlfdmplppdphlhhcf" pwa-extension-url-root="chrome-extension://npnbdojkgkbcdfdjlfdmplppdphlhhcf/" style="margin-left: 380px;">

3. Multiple Form Label:
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">

<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">

4. Empty Button:
<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode">
<span class="headerbtn__icon-container">
<i class="fas fa-expand"></i>
</span>
</button>
```

### Your proposed HTML (or CSS), after change

```html
1. Empty Form Label: To fix this we have to pass aria-label to the input field and remove the label tag but the styling will have to be adjusted for alignment.

<input class="toctree-checkbox" id="toctree-checkbox-1" name="toctree-checkbox-1" type="checkbox" aria-label="toctree-checkbox-1" style="">

2. Missing or Invalid Language: Identify the document language using the <html lang> attribute with a valid value (e.g., <html lang="en">). Ensure that all lang attribute values are valid.

<html lang="en" pwa-launched="true" pwa-extension-id="npnbdojkgkbcdfdjlfdmplppdphlhhcf" pwa-extension-url-root="chrome-extension://npnbdojkgkbcdfdjlfdmplppdphlhhcf/" style="margin-left: 380px;">

3. Multiple Form Label: since we already have the aria-label in he input field we could remove the other label since it is not visible and keep only the input field.

<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">

<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">

4. Empty Button: This can fixed by putting either title or aria-label but since we already have a data-original-title specify in the button element i will go with aria-label.

<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" aria-label="toggle-fullscreen" data-placement="bottom" data-original-title="Fullscreen mode">
<span class="headerbtn__icon-container">
  <i class="fas fa-expand"></i>
  </span>
</button>
```

### More info

No response

**Comment by minrk** [comment]
Thanks for these suggestions and their explanation!

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
